### ISSUE TITLE
[SF_ACC][ui5-button]: Accessibility issue SVG element inside icon only button must have an alternative text 

### ISSUE THREAD

**Issue by Brahmansh** [SAP/ui5-webcomponents#6189]
### **Bug Description**
Accessibility issue: SVG element inside icon only button must have an alternative text. 

### **Expected Behavior**
Either SVG element should have the same accessible text as parent button element OR aria-hidden: true for the SVG element so that it won't be flagged.

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/Button/
2. Enable 'Axe' in dev tools
3. Run scan for the whole page
4. See error "<svg> elements with an img role must have an alternative text"

### **Context**
- UI5 Web Components version: {...}
- OS/Platform: {...}
- Browser: {...}
- Affected component: {...}

### **Log Output / Stack Trace / Screenshots**
Please look at the screenshots -
<img width="1788" alt="Screenshot 2022-12-14 at 2 22 23 PM" src="https://user-images.githubusercontent.com/8809747/207731194-f9e366b7-1a26-4c21-bb72-67d143ccd0bd.png">

<img width="1792" alt="Screenshot 2022-12-14 at 2 25 57 PM" src="https://user-images.githubusercontent.com/8809747/207731216-29f799fa-8897-4c6f-aff1-1343d7e18a8d.png">

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SFSF
- Business impact: {...}

**Comment by plamenivanov91** [comment]
Hello @SAP/ui5-webcomponents-topic-b,

This issue of the reporter can be reproduced easily with a quick scan of axe DevTools in Chrome.

There are a couple of possible solutions described in the reported error in the tool.

Can we fix this?

Regards,
Plamen Ivanov

**Comment by unazko** [comment]
Hello @Brahmansh,

Currently we are working on the following component enhancement
https://github.com/SAP/ui5-webcomponents/issues/5733
We are expecting to complete it in two weeks from now and this would resolve the described issue.

Best Regards,
Boyan

**Comment by vidyakore** [comment]
[Internal Consumption Of SAP]

Hi @unazko and @ilhan007, I am also facing the same problem.
I am working with web components react for SAP internal product, and we are having this accessibility issue.
could you please give update on this issue.

Thanks.
Vidya
